### DISCUSSION TITLE
Question about manifest.py

### DISCUSSION THREAD

**Discussion by gampam2000** [micropython/micropython#15691] [General]
Hi,

I use a custom board definition based on [micropython-example-boards](https://github.com/micropython/micropython-example-boards). Compilation works, but when I want to include/freeze a custom package it does not work. 

for example in my `manifest.py`:
- `module("bh1750.py", base_path="../../lib/bh1750/bh1750")` freezes the module correctly 
- `package("bh1750", base_path="../../lib/bh1750/bh1750") ` **does not work**

the `/lib/bh1750/bh1750/ `directory consists of:
- `__init__.py`
- `bh1750.py`

I need this to work for other packages with more files.

I have read the manifest.py documentation, but maybe I missed something.  Why is `package` not working?
Any help is appreciated. Thanks in advance.

**Comment by shariltumin** [comment]
You might want to take a look at the source in the micropython/extmod/asyncio directory. The manifest.py file there declares asyncio as a package.

**Reply by gampam2000** [reply]
Thanks for the info. It seem the path was wrong:

- `package("bh1750", base_path="../../lib/bh1750/bh1750") `does not work but

- `package("bh1750", base_path="../../lib/bh1750")` works. So it seems it searches for the "package" `bh1750` folder in the base_path.

The documentation https://docs.micropython.org/en/latest/reference/manifest.html#high-level-functions could be a little clearer on that.

**Comment by gampam2000** [comment]
fixed
